applications/fishing-map/src/features/dataviews/dataviews.slice.ts

```diff
@@ -51,4 +51,6 @@ export const selectDataviewById = memoize((id: string) =>
   createSelector([(state: RootState) => state], (state) => selectById(state, id))
 )
 
+export const selectDataviewsStatus = (state: RootState) => state.datasets.status
```

Comment:
I might be lacking context here, but shouldn't it be:
```suggestion
export const selectDataviewsStatus = (state: RootState) => state.dataviews.status
```

Reply:
ouch! 🤦